src/server/api/controllers/users.controller.js

```diff
@@ -31,11 +31,12 @@ const editUser = async (userId, updatedUser) => {
   return knex('users')
     .where({ id: userId })
     .update({
-      title: updatedUser.title,
-      startDate: moment(updatedUser.startDate).format(),
-      endDate: moment(updatedUser.endDate).format(),
-      classId: updatedUser.classId,
-      updatedAt: moment().format(),
+      full_name: updatedUser.full_name,
+      position: updatedUser.position,
+      linkedin: updatedUser.linkedin,
```

Comment:
What if i only wanted to update the position and none of the other columns. Then this will throw an error right!? Because fx updatedUser.position key does not exists